chainer/functions/loss/mean_absolute_error.py

```diff
@@ -19,22 +19,22 @@ def check_type_forward(self, in_types):
         )
 
     def forward_cpu(self, inputs):
-        x0, x1 = inputs
-        self.diff = x0 - x1
-        diff = self.diff.ravel()
+        self.retain_inputs((0, 1))
+        diff = (inputs[0] - inputs[1]).ravel()
```

Comment:
Please keep `x0, x1 = inputs` as unpacking is faster and safer than index access.

Reply:
I removed them for the consistency with `MeanSquaredError`. OK, I'll restore them.